lib/bloc/choice_bloc/choice_bloc.dart

```diff
@@ -19,7 +23,9 @@ class ChoiceBloc extends Bloc<ChoiceEvent, ChoiceState> {
       choice: AppTexts.trackMyPeriod,
     );
 
-    emit(ChoiceNavigateToDateOfBirthPage(summary: summary));
+    summaryRepository.summaryModel = summary;
```

Comment:
Repository is never used for storing data.
Events / States - yes.
Or, this is a result model? if so then please use it as a value of event - state received / emitted instances.